clients/python/README.md

```diff
@@ -56,11 +56,40 @@ Owner and controller of cells in the game. As a programmer of an AI, you are a
             and a dumb AI will take over the player's cells until the player
             becomes active once again.
 
+### Cell
```

Comment:
Do we want to segregate it? I'm feeling like the global player doc will be similar to this one.. Would you only document the functions in this document and the concept of the game in a global player doc?